src/ui/storyTellingServerUI.js

```diff
@@ -147,7 +147,7 @@ sjrk.storyTelling.loadCustomThemeFiles = function (clientConfig) {
 
 
     $.getScript(scriptUrl, function () {
-        window.customThemeScriptLoaded = true;
+        sjrk.storyTelling.customThemeScriptLoaded = true;
```

Comment:
As I mentioned in the other comment, the suggestion was purely for testing and not mean to change things in the actual files.

Reply:
Although going along with your other comment about the callback firing when the script is loaded but not executed, what is actually coming from these JS files? Will you have any issue of trying to access something that hasn't yet been executed?

Reply:
The usual contents of these custom JS files will be grades that are set up as extensions of the page grades, and they will be "instantiated" right after the script is loaded, so it's possible that there may be issues with trying to create a component before the script defining them has been executed. I haven't seen this happen to date.

Reply:
Thinking about this further, the only reliable way I can think of testing (without causing a race condition) that the script was loaded and properly executed would be to mock the server response using Sinon and have the fetched script fire an event for which there is a `listener` test sequence item set up. Otherwise, whether the `loadCustomThemeFiles` promise resolves is all we can reliably determine and depend on, unless of course I'm missing something.

Reply:
From what you've described, I think the race condition could happen in the real world. For example if the processing of the script is slow. I'm guessing that they are small enough that you may not be noticing this. That is, by the time the callback is complete the script has been executed.